tests/pytorch/nightly/hf-lm.libsonnet

```diff
@@ -198,6 +199,7 @@ local utils = import 'templates/utils.libsonnet';
         pip install tensorboardX google-cloud-storage
         echo 'export PATH=~/.local/bin:$PATH' >> ~/.bash_profile
         echo 'export XLA_USE_BF16=1' >> ~/.bash_profile
+        echo 'export XLA_DISABLE_FUNCTIONALIZATION=1' >> ~/.bash_profile
```

Comment:
Can we have a new tpuVm (tpuVmNoFunc) setup that inherits tpuVm and only use that for "hf_lm + v2_8 + roberta_base_pre"?

You can see how pjrt doest it: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/pytorch/nightly/resnet50-mp.libsonnet#L188

@will-cromar Feel free to chime in.

Reply:
Will you be using it in other tests? If it's just for this file, we can add a mixin here that sets this variable, e.g.

```
noFunctionalization:: {
  tpuSettings+: {
    tpuvmExports+: |||
       XLA_DISABLE_FUNCTIONALIZATION=1
    |||
  }
}
```

The correct place to set environment variables for these tests is `tpuVmExports+`. The use of `.bash_profile` is probably just historical baggage.

If it's shared by other tests, feel free to add this field to `common` instead.